jdaviz/tests/test_utils.py

```diff
@@ -14,3 +16,23 @@ def test_alpha_index_exceptions():
         utils.alpha_index(4.2)
     with pytest.raises(ValueError, match="index must be positive"):
         utils.alpha_index(-1)
+
+
+def test_uri_to_download_bad_scheme(imviz_helper):
+    uri = "file://path/to/file.fits"
+    with pytest.raises(ValueError, match='URI file://path/to/file.fits with scheme file'):
+        imviz_helper.load_data(uri)
+
+
+@pytest.mark.remote_data
+def test_uri_to_download_specviz(specviz_helper):
+    uri = "mast:JWST/product/jw02732-o004_t004_miri_ch1-shortmediumlong_x1d.fits"
+    specviz_helper.load_data(uri, cache=True)
+
+
+@pytest.mark.remote_data
+def test_uri_to_download_specviz2d(specviz2d_helper):

```

Comment:
Same comment as above about `local_path`.
```suggestion
def test_uri_to_download_specviz2d(specviz2d_helper, tmp_path):
```